sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -317,6 +297,19 @@ export interface SignedIdentifier {
     id: string;
 }
 
+// @public
+export class TableClient {
+    constructor(url: string, tableName: string, options?: TableServiceClientOptions);
+    createEntity(entity?: Entity, options?: Omit<CreateEntityOptions, "tableEntityProperties">): Promise<CreateEntityResponse>;
```

Comment:
As part of this change you've replaced the use of the `Omit<....>` in the function signature by defining a named type, on the `TableServiceClient`.  I really like this.  Do you plan to do it for the table client as well?

Reply:
Actually, it looks like you might have done this and the report may just be weird because they are not exported from the top level index.ts?